File: Kap14/Python/Listing15/ExceptionExample.py
class MyException(Exception):
    def __init__(self, severity):
        super()
        self.severity = severity

def doSomething():
    raise MyException(1)

try :
    doSomething()
except MyException as mye:
    print("Severity:")
    print(mye.severity)
except Exception as e:
    print("Any other Error!")
finally:
    print("done!")

File: Kap14/JavaScript/Listing14/ExceptionExample.js
class MyError extends Error {
    constructor(severity = 0) {
        super();
        this.severity = severity;
    }
}
  
function doSomething() {
    throw new MyError(1);
}
  
try {
    doSomething();
} catch(e) {
    if (e instanceof MyError) {
        console.log("Severity:" + e.severity);
    } else {
        console.log("Any other Error!")
    }
}
